master.css

```diff
@@ -0,0 +1,24 @@
+body {
```

Comment:
Now make it pretty, add a laugh track when you click the punchline. While the Seinfeld theme song plays on the background.